x-pack/plugins/spaces/public/share_saved_objects_to_space/components/share_to_space_flyout.tsx

```diff
@@ -207,17 +208,23 @@ export const ShareSavedObjectsToSpaceFlyout = (props: Props) => {
     const activeSpace = spaces.find((x) => x.isActiveSpace)!;
     const showShareWarning =
       spaces.length > 1 && arraysAreEqual(currentNamespaces, [activeSpace.id]);
+    const { application, docLinks } = coreStart!;
+    const { Provider: KibanaReactContextProvider } = createKibanaReactContext({
+      application,
+      docLinks,
+    });
```

Comment:
@legrego I made the changes based on https://github.com/elastic/kibana/pull/76132#discussion_r498437258, but I still had to use coreStart in the flyout and pass in the `application` and `docLinks` when creating the Kibana React Context here. Still I think it's cleaner than before :+1: